tests/native/binaries/str_model_tests/sym_strncpy_test.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#define LEN 5
+
+int main() {
+	char * A = "src or dst is A";
+	char * NA = "src or dst is not A";
```

Comment:
Did you mean to do something with `A` or `NA` here?

Reply:
Ah I will remove that. I had an if statement that used those char* previously. I must have forgotten to delete that with copy/paste.